qa/rolling-upgrade/src/javaRestTest/java/org/elasticsearch/upgrades/LogsdbIndexingRollingUpgradeIT.java

```diff
@@ -233,7 +235,13 @@ void query(String dataStreamName) throws Exception {
     protected static void startTrial() throws IOException {
         Request startTrial = new Request("POST", "/_license/start_trial");
         startTrial.addParameter("acknowledge", "true");
-        assertOK(client().performRequest(startTrial));
+        try {
+            assertOK(client().performRequest(startTrial));
```

Comment:
Remove the `assertOk`? I think I was getting it triggering, despite the try-catch..